lib/tilex_web/models/post.ex

```diff
@@ -80,6 +80,27 @@ defmodule Tilex.Post do
     |> hd
   end
 
+  def create_changeset(params, developer_id: developer_id) do
+    %__MODULE__{}
+    |> cast(params, ~w(title body channel_id)a)
+    |> change(developer_id: developer_id)
+    |> add_slug
+    |> validate_required(~w(title body channel_id developer_id)a)
+    |> validate_length(:title, max: title_max_chars())
+    |> validate_length_of_body
+    |> validate_number(:likes, greater_than: 0)
+  end
```

Comment:
Looks like you haven't removed the old `changeset` function, it is still needed?

Reply:
The function is still being used elsewhere in the app